frontend/src/routes/__root.tsx

```diff
@@ -9,5 +12,25 @@ export const Route = createRootRoute({
       <TailwindIndicator />
       <TanStackRouterDevtools />
     </main>
-  )
+  ),
+  notFoundComponent: withAuthRedirect(NotFound)
 });
+
+function NotFound() {
+  return (
+    <>
+      <title>404 - page not found | ZaneOps</title>
+      <div className="flex flex-col gap-5 h-screen items-center justify-center">
+        <Logo className="flex" />
```

Comment:
```suggestion
        <Logo className="md:flex" />
```

Pass the whole className for this component here